bigbone/src/main/kotlin/social/bigbone/api/method/ConversationMethods.kt

```diff
@@ -0,0 +1,51 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.api.Pageable
+import social.bigbone.api.Range
+import social.bigbone.api.entity.Conversation
+
+/**
+ * Direct conversations with other participants. (Currently, just threads containing a post with "direct" visibility.)
+ * @see <a href="https://docs.joinmastodon.org/methods/conversations/">Mastodon conversations API methods</a>
+ */
+class ConversationMethods(private val client: MastodonClient) {
+    /**
+     * View your direct conversations with other participants.
+     * @param range optional Range for the pageable return value
+     * @see <a href="https://docs.joinmastodon.org/methods/conversations/#get">Mastodon API documentation: methods/conversations/#get</a>
+     */
+    @JvmOverloads
+    fun getConversations(range: Range = Range()): MastodonRequest<Pageable<Conversation>> {
+        return client.getPageableMastodonRequest(
+            endpoint = "api/v1/conversations",
+            method = MastodonClient.Method.GET,
+            parameters = range.toParameters()
+        )
+    }
+
+    /**
+     * Removes a conversation from your list of conversations.
+     * @param conversationId ID of the conversation.
+     * @see <a href="https://docs.joinmastodon.org/methods/conversations/#delete">Mastodon API documentation: methods/conversations/#delete</a>
+     */
+    fun deleteConversation(conversationId: String): MastodonRequest<Conversation> {
```

Comment:
Endpoint is defined as returning "empty object", which was implemented as the method having no return value at all in other cases (see `FilterMethods.deleteFilter`, for example). Either this should be the same (no return value, calling `client.performAction(...)` instead), or all other methods should be changed to align them with this, whatever works best.

Doing all like this means we could get rid of `client.performAction(...)` altogether, which _is_ kind of a special case.